x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/config/Job.java

```diff
@@ -98,8 +98,10 @@ public class Job implements SimpleDiffable<Job>, Writeable, ToXContentObject {
     public static final ParseField RESULTS_FIELD = new ParseField("jobs");
 
     // These parsers follow the pattern that metadata is parsed leniently (to allow for enhancements), whilst config is parsed strictly
-    public static final ObjectParser<Builder, Void> LENIENT_PARSER = createParser(true);
-    public static final ObjectParser<Builder, Void> STRICT_PARSER = createParser(false);
+    public static final ObjectParser<Builder, Void> LENIENT_PARSER = createParser(true, true);
+    public static final ObjectParser<Builder, Void> STRICT_PARSER = createParser(true, false);
```

Comment:
Is the `STRICT_PARSER` still used or has it been entirely replaced by `REST_REQUEST_PARSER` ?